docs/sources/tempo/shared/metamonitoring.md

```diff
@@ -0,0 +1,202 @@
+---
+headless: true
+description: Shared file for metamonitoring GET and Tempo.
+labels:
+  products:
+    - enterprise
+    - oss
+---
+
+[//]: # 'This file documents the best practices for monitoring for Tempo.'
+[//]: # 'This shared file is included in these locations:'
+[//]: # '/tempo/docs/sources/tempo/operations/monitor/set-up-monitoring.md'
+[//]: #
+[//]: # 'If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.'
+[//]: # 'Any links should be fully qualified and not relative: /docs/grafana/ instead of ../grafana/.'
+
+
+Metamonitoring for Tempo is handled by the [Grafana Kubernetes Helm chart](https://github.com/grafana/k8s-monitoring-helm) (>=v2.1). Metamonitoring can be used with both microservices and single binary deployments of Tempo.
+
+The Helm chart configures Grafana Alloy to collect metrics and logs.
+
+## Steps
+
+This procudure uses the [Grafana Kubernetes Helm chart](https://github.com/grafana/k8s-monitoring-helm) and the `values.yml` file sets parameters in the Helm chart.
+
+1. Add the Grafana Helm Chart repository, or update, if already added. 
+
```

Comment:
Should this example be indented underneath? OTherwise, it will probably reset the numbered list.